### PR TITLE
Added Liked page with Functionality

### PR DESCRIPTION
**Pull request by rahulcs754**
1. added liked video in liked page
2. remove liked video from listing page
3. remove liked video from liked page

### CODE REVIEW

File: src/Reducer/LikedReducer.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        };
+      }
+
+    case "REMOVE_FROM_LIKED":
+      console.log(action);
+      return {
+        ...state,
+        liked: state.liked.filter((p) => p._id !== action.payload),
+      };
+
```
Comment (raman1999) [inline src/Reducer/LikedReducer.js:8]: no need of remove_from_like, directly can use return data of post api call of like videos and can directly call SET_LIKES

**Review by raman1999 — COMMENTED** [review]
code looks clean to me, suggested few changes

**Review by coldpigli — APPROVED** [review]
Eveything looks great. Naming conventions could have been better. But everything else looks good,

File: src/Reducer/LikedReducer.js (hunk 1 above)
Comment (rahulcs754) [reply src/Reducer/LikedReducer.js:8]: thanks for pointing out . This will change in next commit

Comment (rahulcs754) [reply src/Reducer/LikedReducer.js:8]: done

**Review by raman1999 — COMMENTED** [review]
looks good now

**Pull request merged**